ext/standard/basic_functions.c

```diff
@@ -1237,6 +1238,11 @@ PHP_FUNCTION(time_sleep_until)
 		RETURN_FALSE;
 	}
 
+	if (UNEXPECTED(!(target_secs >= 0 && target_secs <= top_target_sec))) {
+		zend_argument_value_error(1, "must be between 0 and " ZEND_LONG_FMT, (uint64_t)top_target_sec);
```

Comment:
```suggestion
		zend_argument_value_error(1, "must be between 0 and " PRIu64, (uint64_t)top_target_sec);
```

because zend_long isn't always 64 bit.

Reply:
You forgot to add the percentage sign in the last commit.